Dockerfile

```diff
@@ -6,12 +6,18 @@ WORKDIR /sistema
 
 # Copy package.json and package-lock.json
 COPY package.json package-lock.json ./
+COPY prisma ./prisma
 
 # Install dependencies using npm
 RUN npm install
+COPY .env* ./
+
+# Copy the startup script
+COPY start.sh ./
+RUN chmod +x start.sh
 
 # Expose the port the app runs on
 EXPOSE 3000
 
-# Start the Next.js application in development mode
-CMD ["npm", "run", "dev"]
\ No newline at end of file
+# Use the startup script as the entry point
+ENTRYPOINT ["/bin/sh", "./start.sh"]
```

Comment:
Why can't we run the commands in the shell script directly in the Dockerfile?

Reply:
patched